tests/models/patchtsmixer/test_modeling_patchtsmixer.py

```diff
@@ -0,0 +1,973 @@
+            # inputs_dict.pop("target_values")
+        elif model_class in get_values(MODEL_FOR_TIME_SERIES_REGRESSION_MAPPING):
+            rng = random.Random(self.model_tester.seed_number)
+            labels = floats_tensor([self.model_tester.batch_size, self.model_tester.n_targets], rng=rng)
+            # inputs_dict["labels"] = labels
+            inputs_dict["target_values"] = labels
+            # inputs_dict.pop("target_values")
+        elif model_class in [PatchTSMixerModel, PatchTSMixerForPretraining]:
+            inputs_dict.pop("target_values")
+
+        inputs_dict["output_hidden_states"] = True
+        return inputs_dict
+
+    def test_save_load_strict(self):
+        config, _ = self.model_tester.prepare_config_and_inputs()
+        for model_class in self.all_model_classes:
+            model = model_class(config)
+
+            with tempfile.TemporaryDirectory() as tmpdirname:
+                model.save_pretrained(tmpdirname)
+                model2, info = model_class.from_pretrained(tmpdirname, output_loading_info=True)
+            self.assertEqual(info["missing_keys"], [])
+
+    def test_hidden_states_output(self):
+        def check_hidden_states_output(inputs_dict, config, model_class):
+            model = model_class(config)
+            model.to(torch_device)
+            model.eval()
+
+            with torch.no_grad():
+                outputs = model(**self._prepare_for_class(inputs_dict, model_class))
+
+            hidden_states = outputs.encoder_hidden_states if config.is_encoder_decoder else outputs.hidden_states
+
+            expected_num_layers = getattr(
+                self.model_tester,
+                "expected_num_hidden_layers",
+                self.model_tester.num_hidden_layers,
+            )
+            self.assertEqual(len(hidden_states), expected_num_layers)
+
+            expected_hidden_size = self.model_tester.hidden_size
+            self.assertEqual(hidden_states[0].shape[-1], expected_hidden_size)
+
+            num_patch = self.model_tester.num_patches
+            self.assertListEqual(
+                list(hidden_states[0].shape[-2:]),
+                [num_patch, self.model_tester.hidden_size],
+            )
+
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+
+        for model_class in self.all_model_classes:
+            # inputs_dict["output_hidden_states"] = True
+            print("model_class: ", model_class)
+
+            check_hidden_states_output(inputs_dict, config, model_class)
+
+            # check that output_hidden_states also work using config
+            # del inputs_dict["output_hidden_states"]
+            # config.output_hidden_states = True
+
+            # check_hidden_states_output(inputs_dict, config, model_class)
+
+    @unittest.skip("No tokens embeddings")
+    def test_resize_tokens_embeddings(self):
+        pass
+
+    def test_model_outputs_equivalence(self):
```

Comment:
This test should pass IMO and should not be skipped? Being able to return tuples is VERY useful for `torch.compile` etc...

Reply:
https://github.com/huggingface/transformers/pull/26247/commits/3daac8886a75ab9c285c172c5bd5e6073973c4aa